src/main/resources/static/css/typography.css

```diff
@@ -0,0 +1,54 @@
+a {
+    text-decoration-color: white;
+    font-size: 10px;
+    text-transform: uppercase;
+}
+
+.site-header a {
+    color: white;
+    transition: ease-in-out color .15s;
+    font-size: 20px;
+}
+
+.site-header a:hover {
+    color: #fff;
+    text-decoration: none;
+}
+
+h1 {
+    margin-top: 20px;
+}
+
+label {
+    font-size: 18px;
+}
+
+.error-message {
+    font-size: 18px;
+    margin-top: 30px;
+    text-align: center;
+}
+
+li {
+    color: #0000ff;
+}
+
+div.active a {
+    color: #6ed2f0;
+}
+
+.poppins-semibold {
+    font-family: "Poppins", sans-serif;
+    font-weight: 600;
+    font-style: normal;
+}
+
+body {
+    background-color: whitesmoke;
```

Comment:
ok maybe that was confusing...the background color is NOT typography - so it should be in the main style file still. but the body gets a typography - thus can be here (i would put it all the way up though)